organise_desktop/organiseDesktop.py

```diff
@@ -221,7 +237,7 @@ def undo():
 
     # list of folders to be created
     elif sys.platform == 'linux' or sys.platform == 'darwin':
-        if environ.get('TEST_DIR') != '':
+        if environ.get('TEST_DIR'):
```

Comment:
Would it be better to actually explicitly test this is `not None`?

Reply:
Oh, Yes. I didn't think about that.

I'll start working on it tomorrow, I'm about to sleep right now.

Thanks for the review.

Reply:
Jesus, It feels like I've been asleep for over a month. I completely forgot about this.

I pushed a new commit fixing that - IIRC That part was causing issue because it was None, not string. so It will gonna work.